src/network/hub.rs

```diff
@@ -1073,6 +1171,14 @@ impl HubConn {
         self.send_just_saying("free_joints_end", Value::Null)?;
         Ok(())
     }
+
+    fn send_stored_device_messages(
+        &self,
+        _db: &Connection,
+        _device_address: &String,
+    ) -> Result<()> {
+        unimplemented!()
```

Comment:
currently we don't support message send, just return Ok should be fine, we should have another issue to track text message support